### PR TITLE
Run pre-commit to validate PR in main branch

### PR DESCRIPTION
**Pull request by harshthakkar01**
### Submission Checklist

Please take the following actions before submitting this pull request.

* Fork your PR branch from the Toolkit "develop" branch (not main)
* Test all changes with pre-commit in a local branch [#](https://goo.gle/hpc-toolkit#development)
* Confirm that "make tests" passes all tests
* Add or modify unit tests to cover code changes
* Ensure that unit test coverage remains above 80%
* Update all applicable documentation
* Follow Cloud HPC Toolkit Contribution guidelines [#](https://goo.gle/hpc-toolkit-contributing)

### CODE REVIEW

File: .github/workflows/pr-precommit.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@
 #     http://www.apache.org/licenses/LICENSE-2.0
 #
 # Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
+# distributed under the License is distributed on an "AS IS" BASIS,`
```
Comment (nick-stroud) [inline .github/workflows/pr-precommit.yml:10]: nit: why this change?

**Review by nick-stroud — APPROVED** [review]

File: .github/workflows/pr-precommit.yml (hunk 1 above)
Comment (harshthakkar01) [reply .github/workflows/pr-precommit.yml]: :-)
